addons/website/tools.py

```diff
@@ -22,7 +23,7 @@ def MockRequest(
         # website_sale
         sale_order_id=None, website_sale_current_pl=None,
 ):
-
+    from odoo.tests.common import HttpCase  # noqa: PLC0415
```

Comment:
Ok, but would be cleaner to move MockRequest outside this file and add it in a specific file.

Reply:
Added a TODO in the code for this.